Sheppard/src/main/java/com/truex/sheppard/MainActivity.java

```diff
@@ -272,4 +280,42 @@ private void setupDataSourceFactory() {
         String userAgent = Util.getUserAgent(this, applicationName);
         dataSourceFactory = new DefaultDataSourceFactory(this, userAgent, null);
     }
+
+    private void setupIntents() {
+        registerReceiver(hdmiStateChange, new IntentFilter(INTENT_HDMI));
+        registerReceiver(audioWillBecomeNoisy, new IntentFilter(INTENT_NOISY_AUDIO));
```

Comment:
is the audioWillBecomeNoisy one a requirement? It could be not because of network conditions, in which case one wants to still play the video as best it can?

We have never had to worry about this issue with any androidtv/firetv apps we have made before.